tests/unittests/test_enable_debug_logging_functions.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import typing
+import os
+from unittest.mock import patch
+
+from azure_functions_worker import testutils
+from azure_functions_worker.constants import PYTHON_ENABLE_DEBUG_LOGGING
+
+
+class TestDebugLoggingEnabledFunctions(testutils.WebHostTestCase):
```

Comment:
Checking - didn't you have to update the host.json file for enabling the logging? Or is it enabled by default now?

If yes - Do we need another test where it is not enabled by default and we are still enabling the flag and checking that the new debug log doesn't get logged?

Ideally - there is a potential issue - 
If a customer didn't update host.json, but enables the new feature flag, we start sending the logs from our end. This could result the choking of the gRPC channel that we already know about. If we can also get the host.json information from the host (which I think we do get during worker_init), then we can also do some check if we need to actually send the logs at all or not.
*Note*: Not very important but something to be aware of.

Reply:
Good catch, will add a test case for that. As discussed, the host.json logging filter is set at function level while this flag is at function app level for the shared root logger. It is not possible to define the flag validity if functions have inconsistent log levels in the host.json. Separate investigation/change will be made to warn customers in case of inconsistent host.json log level and PYTHON_ENABLE_DEBUG_LOGGING value being set.